### PR TITLE
Optimized Option model queries

### PR DESCRIPTION
**Pull request by jerodev**
- Select only `value` in `get()` function
- Use `updateOrCreate` to create or update values
- Delete records directly without fetching them first

### CODE REVIEW

File: src/Option.php [hunk 1]
```diff
@@ -62,26 +62,18 @@ public function set($key, $value = null)
         $keys = is_array($key) ? $key : [$key => $value];
 
         foreach ($keys as $key => $value) {
-            $option = self::firstOrNew(['key' => $key]);
-
-            $option->value = $value;
-
-            $option->save();
+            self::firstOrCreate(['key' => $key], ['value' => $value]);
```
Comment (gizburdt) [inline src/Option.php]: This will not work when updating an existing option

Comment (jerodev) [reply src/Option.php]: You are correct, I used the wrong function.
I have created an edit where I replaced the function with `updateOrCreate`.

Comment (gizburdt) [reply src/Option.php]: Awesome!

**Comment by gizburdt** [comment]
Thanks! Will merge and tag 2.2 :)

File: src/Option.php [hunk 2]
```diff
@@ -43,7 +43,7 @@ public function exists($key)
      */
     public function get($key, $default = null)
     {
-        if ($option = self::where('key', $key)->first()) {
+        if ($option = self::where('key', $key)->select('value')->first()) {
```
Comment (lucasmichot) [inline src/Option.php:46]: This introduces a bug @jerodev . Can you fix your code?

**Comment by lucasmichot** [comment]
@gizburdt can you enable TravisCI before merging this?
Because this PR introduces a bug and tests fail

**Comment by gizburdt** [comment]
Will do! Thanks for the heads up :)

**Comment by gizburdt** [comment]
Travis is failing big time. I don't have much experience with it.. Could you help set this up @lucasmichot ?

**Comment by lucasmichot** [comment]
@gizburdt `.travis.yml` is already set-up in your repository.
You have to enable buils on this repo in https://travis-ci.org/

**Comment by gizburdt** [comment]
Travis is already enabled, but failing

**Comment by gizburdt** [comment]
@lucasmichot Could you help us further with this issue, please? :)

**Pull request merged**